CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.0.1
```

Comment:
Probs just a typo, but just to catch this in case we release with the wrong version number... is should probably be `Head` or `4.1.1`.